miasm/arch/mips32/sem.py

```diff
@@ -488,6 +509,66 @@ def ei(arg1):
 def ehb(arg1):
     "NOP"
 
+@sbuild.parse
+def sc(arg1, arg2):
+    arg1 = arg2;
+    arg2 = ExprInt(0x1, 32)
+
+@sbuild.parse
+def mthi(arg1):
+    R_HI = arg1
+
+@sbuild.parse
+def mtlo(arg1):
+    R_LOW = arg1
+
+@sbuild.parse
+def clz(rs, rd):
```

Comment:
Maybe you can use the miasm operator "cntlz". For example, in arm, we have:
```python
def clz(ir, instr, a, b):
    e = []
    e.append(ExprAssign(a, ExprOp('cntleadzeros', b)))
    return e, []
```

Reply:
I have replace it with 'cntleadzeros'

Reply:
However, a "manual" implementation of clz (from which I have copied my past implementation) it is present in arch/mep/sem.py

Reply:
Oh! I think I did a bad review of the mep before merging it.